srv/components/provisioner/scripts/cortx_prep.sh

```diff
@@ -0,0 +1,291 @@
+            echo "ERROR: minion srvnode-0 seems still not ready after $max_tries attempts." >> ${LOG_FILE}
+            echo "ERROR: Provisioner configuration manager failed to connect to local server" | tee -a ${LOG_FILE}
+            exit 1
+        fi
+        try=$(( $try + 1 ))
+    done
+    echo "Minion on Server A started successfully" >> ${LOG_FILE}
+    echo "Done" | tee -a ${LOG_FILE}
+}
+
+create_factory_user()
+{
+    user_nodeadmin="nodeadmin"
+    user_support="support"
+
+    echo "Creating user: $user_nodeadmin" | tee -a ${LOG_FILE}
+    #groupadd -g ${gid} ${group}
+    useradd -c "$user_nodeadmin" -d /home/${user_nodeadmin}
+    echo "Done" | tee -a ${LOG_FILE}
+
+    echo "Creating user: $user_support" | tee -a ${LOG_FILE}
+    #groupadd -g ${gid} ${group}
+    useradd -c "$user_suport" -d /home/${user_support}
+    echo "Done" | tee -a ${LOG_FILE}
+
+}
+
+setup_repos()
+{
+    #repo="http://cortx-storage.colo.seagate.com/releases/cortx/github/main/centos-7.8.2003/1291/prod/"
+
+    repo=$1
+    yum install -y yum-utils
+    echo "Configuring the repository: ${repo}/3rd_party" | tee -a ${LOG_FILE}
+    yum-config-manager --add-repo "${repo}/3rd_party/"
+    echo "Configuring the repository: ${repo}/cortx_iso" | tee -a ${LOG_FILE}
+    yum-config-manager --add-repo "${repo}/cortx_iso/"
+
+    yum clean all
+}
+
+install_cortx_pkgs()
+{
+
+    echo "Installing Cortx packages" | tee -a ${LOG_FILE}
+    cortx_pkgs=(
+        "java-1.8.0-openjdk-headless"
+        "cortx-cli.x86_64"
+        "python3 sshpass"
+        "python36-m2crypto"
+        "salt"
+        "salt-master"
+        "salt-minion"
+        "cortx-prereq"
+        "python36-cortx-prvsnr"
+        "cortx-csm_agent.x86_64"
+        "cortx-csm_web.x86_64"
+        "cortx-ha.x86_64"
+        "cortx-hare.x86_64"
+        "cortx-libsspl_sec.x86_64"
+        "cortx-libsspl_sec-method_none.x86_64"
+        "cortx-libsspl_sec-method_pki.x86_64"
+        "cortx-motr.x86_64"
+        "cortx-motr-ivt.x86_64"
+        "cortx-prereq.x86_64"
+        "cortx-prvsnr.x86_64"
+        "cortx-prvsnr-cli.x86_64"
+        "cortx-py-utils.noarch"
+        "cortx-s3server.x86_64"
+        "cortx-sspl.noarch"
+        "cortx-sspl-cli.noarch"
+        "python36-cortx-prvsnr.x86_64"
+        "stats_utils.x86_64"
+        "udx-discovery.x86_64"
+    )
+
+    for pkg in ${cortx_pkgs[@]}; do
+        echo "Installing $pkg" | tee -a ${LOG_FILE}
+        rpm --quiet -qi $pkg ||  yum install --nogpgcheck -y -q $pkg  && \
+            echo "Package $pkg is already installed."
+    done
+
+    echo "Installing nodejs" | tee -a ${LOG_FILE}
+    echo "Downloading nodeja tarball" >> ${LOG_FILE}
+    wget ${nodejs_tar}
+    echo "Extracting the tarball" >> ${LOG_FILE}
+    mkdir /opt/nodejs
+    tar -C /opt/nodejs/ -xvf node-v12.13.0-linux-x64.tar.xz
+    echo "The extrracted tarball is kept at /opt/nodejs, removing the tarball ${nodejs_tar}" >> ${LOG_FILE}
```

Comment:
Make this debug log, fix typo spelling errors

Reply:
It's debug only, redirected to LOG_FILE directly, added 'DEBUG' keyword now